horace_core/sqw/@main_header_cl/main_header_cl.m

```diff
@@ -197,48 +254,49 @@
             % creation date becomes "known";
             obj = set_filename_with_cdate_(obj,val);
         end
+
         %------------------------------------------------------------------
+
         function ver  = classVersion(~)
             % define version of the class to store in mat-files
             % and nxsqw data format. Each new version would presumably read
             % the older version, so version substitution is based on this
             % number
             ver = 1;
         end
+
         function flds = saveableFields(~)
             flds = main_header_cl.fields_to_save_;
         end
-        %------------------------------------------------------------------
+
     end
+
     methods(Static)
         % Service routines:
         function datt = convert_datetime_from_str(in_str)
             % convert main_header_cl's generated time string into
-            % datetime format (standard datetime conversion works very
-            % strangely)
-            val = num2cell(sscanf(in_str,main_header_cl.DT_format_));
-            datt = datetime(val{:});
+            % datetime format
```

Comment:
Remembering your comment on this and appreciating that the original strangely comment is opaque. But we have now lost a warning that something may go wrong if the standard conversion is used. Maybe stating what the standard conversion is and advising it has had previous problems?